pennylane/vqe/vqe.py

```diff
@@ -547,7 +572,7 @@ def __init__(
         self._multiple_devices = isinstance(device, Sequence)
         """Bool: Records if multiple devices are input"""
 
-        if all(c == 0 for c in coeffs) or not coeffs:
+        if np.isclose(qml.math.toarray(qml.math.count_nonzero(coeffs)), 0):
```

Comment:
`count_nonzero` works in all interfaces with no changes?? 🤯

Reply:
Ehm, it seems so? :)